postcss.config.js

```diff
@@ -1,5 +1,5 @@
 const autoprefixerOptions = [
-    '> 1%',
+    '>2%',
```

Comment:
This is percent usage for a particular browser version? Is that right?

Reply:
Yes ... it is directly from the guidance on the USWDS web site